src/worker/network.rs

```diff
@@ -381,8 +398,10 @@ impl Worker for NetworkWorker {
             target_port,
             arrival_rate: _,
             departure_rate: _,
-            nconnections: _,
+            n_static_connections: _,
+            n_dyn_connections: _,
             send_interval: _,
+            preempt: _,
```

Comment:
We might want to use `..` here
https://doc.rust-lang.org/book/ch19-03-pattern-syntax.html#ignoring-remaining-parts-of-a-value-with-